88.md

```diff
@@ -0,0 +1,131 @@
+# NIP-88
+
+## Polls
+
+`draft` `optional`
+
+This NIP defines the event scheme that describe Polls on nostr.
+
+## Events
+
+### Poll Event
+
+The poll event is defined as a `kind:1068` event.
+
+- **content** key holds the label for the poll.
+
+Major tags in the poll event are:
+
+- **option**: The option tags contain an OptionId(any alphanumeric) field, followed by an option label field.
+- **relay**: One or multiple tags that the poll is expecting respondents to respond on.
+- **polltype**: can be "singlechoice", "multiplechoice", or "rankedchoice". Polls that do not have a polltype should be considered a "singlechoice" poll.
```

Comment:
I would like to have different kind numbers for each poll type instead of this `polltype` tag, so clients can specifically query the ones they support.

Reply:
The usecase make sense, should we use an indexable tag like ["T", <poll-type>] instead, that way we wouldn't be polluting kinds

Reply:
I'm ok either way but I think having different kinds is cleaner.

Reply:
Yes, multiple types make more sense because you can always expand to new poll types without breaking current clients with new types as tags.